src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -271,799 +269,6 @@ categories :
 === END_MPI_T_CVAR_INFO_BLOCK ===
 */
```

Comment:
I think these CVARs can go with the functions below that use them (which are all now in `ofi_init.c`).

Reply:
Good point. I can do that. IIRC the CVARs generated code is not in the file where the CVARs are defined, right?

Reply:
Right. All of the code goes in `mpir_cvars.h`.